src/Akeneo/Platform/Bundle/ImportExportBundle/Domain/Event/FileCannotBeExported.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Akeneo PIM Enterprise Edition.
+ *
+ * (c) 2022 Akeneo SAS (https://www.akeneo.com)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Akeneo\Platform\Bundle\ImportExportBundle\Domain\Event;
+
+class FileCannotBeExported
```

Comment:
```suggestion
class FileTransferFailed
```
It's just a suggestion.
`FileCannotBeExported` looks like it misses some configuration or data to launch the export

Reply:
FileTransferFailed might be confusing later if we have an another event for file tranfer before import. I let it like that in this PR. 👍🏻

Reply:
I see, it makes sense then!